frontend/src/mixins/apiPropsMixin.js

```diff
@@ -35,6 +35,7 @@ export const apiPropsMixin = {
     /* enable/disable auto save */
     autoSave: { type: Boolean, default: true, required: false },
     autoSaveDelay: { type: Number, default: 800, required: false },
+    cancelable: Boolean,
```

Comment:
```suggestion
    /* for non-autoSave mode: displays cancel button from the beginning (even if not dirty) */
    cancelable: { type: Boolean, default: false, required: false },
```

I understood this might anyway being extracted into a separate PR, right?